### PR TITLE
Add date attribute to Opportunity

### PR DESCRIPTION
**Pull request by Himalee**
When adding a new Opportunity to the database, users can input a date.

### CODE REVIEW

**Comment by ssunday** [comment]
On MSS, for database migrations we tag our commits with `[MIGRATION] Add x to y`

File: src/main/java/PostgresCommunicator.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public PostgresCommunicator(String databaseURL) {
     }
 
     public void writeToDatabase(Opportunity opportunity) throws SQLException, ClassNotFoundException {
-        String sqlQuery = String.format("INSERT INTO OPPORTUNITIES (name, description, proposed_cost, user_name, stage, opportunity_uuid) VALUES ('%s', '%s', %d, '%s', '%s', '%s');", opportunity.getName(), opportunity.getDescription(), opportunity.getProposedCost(), opportunity.getUserName(), opportunity.getStage(), opportunity.getUuid());
+        String sqlQuery = String.format("INSERT INTO OPPORTUNITIES (name, description, proposed_cost, user_name, stage, opportunity_uuid, opportunity_date) VALUES ('%s', '%s', %d, '%s', '%s', '%s', TO_DATE('%s', 'dd/mm/yyyy'));", opportunity.getName(), opportunity.getDescription(), opportunity.getProposedCost(), opportunity.getUserName(), opportunity.getStage(), opportunity.getUuid(), opportunity.getDate());
```
Comment (ssunday) [inline src/main/java/PostgresCommunicator.java:16]: is the to_date necessary? I don't think it needs to—I think YYYY-MM-DD works?

Comment (Himalee) [reply src/main/java/PostgresCommunicator.java:16]: Looks like it is needed as without I get this error message: `org.postgresql.util.PSQLException: ERROR: column "opportunity_date" is of type date but expression is of type record`

Comment (ssunday) [reply src/main/java/PostgresCommunicator.java:16]: Ah

**Pull request merged**